src/components/DropZone.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import '../styles/DropZone.scss'
+
+const DropZone = props => {
+  const { onDrop, onDragLeave, channelName } = props
+  return (
+    <div
+      className="dropzone"
+      onDrop={event => {
```

Comment:
I think it would be simpler if you just pass the *onDrop* as is like I suggested with *onDragLeave* below.